tests/cli/test_correct_scraper.py

```diff
@@ -3,13 +3,23 @@
 import mozfile
 from mozprocess import processhandler
 
+from mozdownload import Scraper
 import mozhttpd_base_test as mhttpd
 
 
+default_latest_binary_names = {
+    'linux': 'firefox-latest-en-US.linux.tar.bz2',
+    'linux64': 'firefox-latest.en-US.linux64.tar.bz2',
+    'mac': 'firefox-latest.en-US.mac.dmg',
+    'mac64': 'firefox-latest.en-US.mac64.dmg',
+    'win32': 'firefox-latest.en-US.win.exe',
+    'win64': 'firefox-latest.en-US.win.exe'
+}
+
 tests = [
     # ReleaseScraper
     {'options': ['-v', 'latest'],
-     'fname': 'firefox-latest.en-US.linux64.tar.bz2'},
+     'fname': default_latest_binary_names[Scraper.detect_platform()]},
```

Comment:
I would suggest that we factor those platform specific tests out into its own test and compare the retrieved filename with the DEFAULT_FILE_EXTENSIONS entry from scraper.py. At the same time we might want to mock the platform so that we can test any of those supported ones.